### ISSUE TITLE
Switch Case Statements will melt to the left with each reformat

### ISSUE THREAD

**Issue by benthillerkus** [millennIumAMbiguity/processing-formatter#4]
**Describe the bug**
When you format the indentation of switch-case statements will be lost

**To Reproduce**
1. Write a switch-case
2. Format it yourself
3. Run the auto formatter

**Expected behavior**
The users indentation should be kept, or better, the switch case should be recognized and the formatting should act accordingly.

**Screenshots**
![2021-05-10_23-35-36](https://user-images.githubusercontent.com/29630575/117727611-8e212d00-b1e8-11eb-8da8-4d224d94a20e.gif)

**Additional context**
I'm using spaces, if that matters.

**Comment by millennIumAMbiguity** [comment]
Thank you for your report, the issue has now been resolved.

**Comment by benthillerkus** [comment]
Hi, thanks again for you fix, but I think it's still somewhat broken, if after a `case` no `break;` follows:
![2021-05-13_19-42-02](https://user-images.githubusercontent.com/29630575/118164388-5a7e1700-b423-11eb-9aa9-8c2a5d736cca.gif)

For context, what I'm doing in the GIF is:
1. I start out with line 10 being badly formatted. There is no `break;`
2. I run the formatter. The formatter keeps the indentation level of the `case` for the rest of the file, presumably because no `break` followed
3. I crtl+z
4. I added the break statement
5. I formatted again. Initially it suffers from the same issues as in 2.
6. I run the formatter again, until the indentation is correct / the result _converges_
